user/templates/templates.md

```diff
@@ -83,6 +83,15 @@ developers do not test them.
 * [Gentoo](/doc/templates/gentoo/)
 * [Gentoo Minimal](/doc/templates/minimal/)
 
+## Windows
+
+Windows templates are constructed differently from Linux-based templates as
+Windows is a closed source system that can be modified only after installing.
+So it is not possible to provide preconfigured Windows templates for Qubes.
+The process of installing a Windows qube and connecting it to the Qubes
+environment via installing Qubes Windows Tools (QWT) is desribed in several
+chapters in [Windows qubes](/doc/templates/windows/windows.md).
```

Comment:
This should point at the page permalink, not md filename. So, `/doc/templates/windows/`

Reply:
Ah, I fixed a ton of these and must have somehow missed this one.